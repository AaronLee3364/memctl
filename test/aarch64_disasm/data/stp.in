fffffff0070c0000 a9be4ff4 a9017bfd a8002468 a8817d3f
