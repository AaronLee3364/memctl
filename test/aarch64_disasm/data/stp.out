fffffff0070c0000  STP     X20, X19, [SP, #-32]!
fffffff0070c0004  STP     X29, X30, [SP, #16]
fffffff0070c0008  STNP    X8, X9, [X3]
fffffff0070c000c  STP     XZR, XZR, [X9], #16
